src/librustc_middle/hir/map/collector.rs

```diff
@@ -325,7 +336,7 @@ impl<'a, 'hir> Visitor<'hir> for NodeCollector<'a, 'hir> {
 
     fn visit_param(&mut self, param: &'hir Param<'hir>) {
         let node = Node::Param(param);
-        self.insert(param.pat.span, param.hir_id, node);
+        self.insert(param.span, param.hir_id, node);
```

Comment:
This change should result in some user-visible result, and somehow does not appear in the tests.

Reply:
Actually, this span was only used inside a debugging operation. So no change in behaviour.